examples/channels-example/README.md

```diff
@@ -108,3 +108,16 @@ Repeat steps 1 to 6.
 1. Second tab: Fill in `Set event type` to `message`
 1. Second tab: Fill in `Event message` with e.g. `{"name":"john","foo":"bar"}`
 1. Second tab: new message should be displayed, First tab: **no new message**
+
+## One word to distributed Tracing
+
+RIG processes distributed tracing context differently depending on the type of a message: sometimes RIG expects the trace context to be in the (HTTP) header, sometimes in the event payload itself.
```

Comment:
that sounds like this would be random :D instead of "sometimes", perhaps use this:

> RIG handles two types of messages: request/response messages (of any format) and events (in CloudEvents format). Tracing the former is done using headers - e.g., for HTTP RIG implements the w3c trace context specification. Tracing CloudEvents is done using the official CloudEvents tracing extension, where parent trace ID and trace context are taken from the context attributes of the event itself rather than from trace context headers.